packages/p2p/src/components/modal-manager/modals/market-rate-change-error-modal/market-rate-change-error-modal.tsx

```diff
@@ -61,6 +65,7 @@ const MarketRateChangeErrorModal = ({ submitForm, values }: TMarketRateChangeErr
                         onClick={() => {
                             hideModal();
                             buy_sell_store.form_props.setIsMarketRateErrorModalOpen(false);
+                            if (is_desktop) buy_sell_store.setTempContactInfo(null);
```

Comment:
only need to hide it for desktop once we click cancel, since it will disappear on responsive view, but it should still be there.